pkg/grub/rootfs.cfg

```diff
@@ -198,6 +198,7 @@ function set_arm64_qemu {
 }
 
 function set_kvm_boot {
+   set_generic
```

Comment:
I don't think you need this here. set_kvm_boot is executed after set_generic

Reply:
Removed.